docs/getting-started/brokers/django.rst

```diff
@@ -37,6 +37,10 @@ configuration values.
 .. code-block:: console
 
     $ python manage.py syncdb
+    
+ Or if you are using Django 1.9 or later
```

Comment:
It should actually, but I'm specifically fixing this because it doesn't work on Django 1.9

I guess using this should be the better option. Maybe better worded as "If you are using Django 1.7 or later you should use:"